galleries.py

```
@@ -512,376 +534,177 @@ def _gallery(fileroot, product_ids, catalog, links,
+                'alt': alt,
+                'title': title
+            }
+            if catalog[id].is_movie:
+                row_data['movie'] = True
+
+            row_data = {k: v for k, v in row_data.items() if v is not None}
+            floated_img_blocks.append({k: v for k, v in row_data.items() if v is not None}) 
+
+        return floated_img_blocks
+
+    def write_yaml_file(filename, yaml_data):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `write_yaml_file`

Add return type annotation: `None`

(ANN202)